plugins/view-transitions/includes/theme.php

```diff
@@ -42,4 +103,44 @@ function plvt_load_view_transitions(): void {
 	wp_register_style( 'wp-view-transitions', false, array(), null ); // phpcs:ignore WordPress.WP.EnqueuedResourceParameters.MissingVersion
 	wp_add_inline_style( 'wp-view-transitions', $stylesheet );
 	wp_enqueue_style( 'wp-view-transitions' );
+
+	$theme_support = get_theme_support( 'view-transitions' );
```

Comment:
Per above, this could be replaced with:

```suggestion
	$theme_support = plvt_get_view_transitions_theme_support();
```